src/Support/Facade.php

```diff
@@ -10,31 +10,4 @@
  */
 class Facade extends FacadeParent
 {
-
-    /**
-     * @inheritDoc
-     */
-    protected static function resolveFacadeInstance($name)
-    {
-        if (
-            !is_object($name) &&
-            !is_null(static::$app) &&
-            !static::$app->bound($name) &&
-            ($instance = static::getFacadeInstance()) !== null
-        ) {
-            static::$app->instance($name, $instance);
-        }
-
-        return parent::resolveFacadeInstance($name);
-    }
-
-    /**
-     * If the accessor is not found via getFacadeAccessor, use this instance as a fallback.
-     *
-     * @return mixed
-     */
-    protected static function getFacadeInstance()
-    {
-        return null;
-    }
```

Comment:
Why were these changes required?

Reply:
As far as I can tell, this method is not used anywhere in the code. Seemed to be a holdover from a previous Laravel version.